src/hotspot/share/gc/serial/serialArguments.cpp

```diff
@@ -28,8 +28,8 @@
 #include "gc/serial/serialArguments.hpp"
 #include "gc/serial/serialHeap.hpp"
 
-void SerialArguments::initialize_heap_flags_and_sizes() {
-  GenArguments::initialize_heap_flags_and_sizes();
+void SerialArguments::initialize() {
+  GenArguments::initialize();
```

Comment:
```suggestion
  GCArguments::initialize();
```
There's no `GenArguments::initialize()` so I think it would be clearer to call the function with the parent class instead. Alternatively, make `ParallelGC::initialize` also call `GenArguments::initialize`.